crates/pallets/starknet/src/tests/utils.rs

```diff
@@ -112,3 +116,58 @@ pub fn build_get_balance_contract_call(account_address: StarkFelt) -> (EntryPoin
 
     (balance_of_selector, calldata)
 }
+
+pub fn create_resource_bounds() -> starknet_api::transaction::ResourceBoundsMapping {
+    let mut map = BTreeMap::new();
+    map.insert(
+        starknet_api::transaction::Resource::L1Gas,
+        starknet_api::transaction::ResourceBounds { max_amount: 10000, max_price_per_unit: 12000 },
+    );
+    map.insert(
+        starknet_api::transaction::Resource::L1Gas,
+        starknet_api::transaction::ResourceBounds { max_amount: 50000, max_price_per_unit: 31000 },
+    );
+    starknet_api::transaction::ResourceBoundsMapping(map)
+}
+
+fn stark_felt_to_field_element(felt: &StarkFelt) -> FieldElement {
+    // Can only fail if the 32 byte number is not smaller than the field's modulus,
+    // but StarkFelt already enforces that
+    FieldElement::from_bytes_be(&felt.0).unwrap()
+}
```

Comment:
Remove this and uses Felt252Wrapper for conversions instead. It implement all the From/Into you need

Reply:
I have set this `FieldElement::from_bytes_be(&felt.0).unwrap()` in my function. The `get_storage_key` func want a `FieldElement`. I have a `ContractAddress`, the From Trait of `Felt252Wrapper` for `PatriciaKey` exist but i can't retrieve PatriciaKey from my ContractAddress but i can retrieve `StarkFelt`. I think i have do the best with existing code, say me if i can do better